src/services/auth/plugins/passport/strategies.ts

```diff
@@ -0,0 +1,21 @@
+export enum PassportStrategy {
```

Comment:
I don't have a strong opinion about that. I think we should just be fine with our choice and all use the same convention. 

I know that java use SCREAMING_CASE but I think PascalCase is used by microsoft so in my opinion we can continue with that. 

https://github.com/microsoft/TypeScript/wiki/Coding-guidelines (I know it is for the contributors of Typescript).
<img width="292" alt="image" src="https://github.com/graasp/graasp/assets/147397675/548fe98e-8730-4621-b5ea-35c3fa0f3004">

This is the same in C# (https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/enum), so if we have mostly all our enums in PascalCase, I think it is okay to keep it like that.

What do you think @Defferrard, @spaenleh, @pyphilia ?

Reply:
Well, I'm frustrated to see enumeration in PascalCase, but the conventions of the langage (Even if it is not destined to us) are a real strong arguments to me. I think we should follow the closest standards we can found. From my researches, every langages and developers have their own convention. I just discovered a whole new world where people just don't use CONSTANT_CASE for enumeration, it's okay. We won't be able to unify everything anyway, since 2 of our libraries use different case...